source/how-tos/integrate-fastboot.rst

```diff
@@ -70,3 +73,57 @@ The following steps provide an example to enable fastboot feature from **USB** k
   Run::
 
     fastboot devices
+
+
+Load from SPI
+^^^^^^^^^^^^^^^^^
+
+You can also add fastboot into SBL image and program it into SPI flash.
+
+
+**Step 1:** Once you created ``iasimage.bin``, copy it into |SPN| directory::
+
+    cp iasimage.bin Platform/ApollolakeBoardPkg/SpiIasBin/iasimage1.bin
+
+**Step 2:** Replace the last boot option to load fastboot from SPI

```

Comment:
Looks like, int he example, you are not replacing last boot option, but just creating one more option at the end of the list.

Reply:
fixed